phylogenetic/defaults/exclude.txt

```diff
@@ -0,0 +1,16 @@
+KAK_428
+NGA/2016/ISTH_0779
+GUINEA_Z_185a
+NIGERIA_IKEJI
```

Comment:
Slight tangent, the addition of the GenBank accessions below made me realize these sequences should probably be updated to their accessions to be excluded.

Reply:
Fixed in https://github.com/nextstrain/lassa/pull/19/commits/ac267a6f3f1415c4206c8e4a7c0372c2d343e524